lib/resque/plugins/timed_round_robin/resque_worker.rb

```diff
@@ -0,0 +1,24 @@
+module Resque
+  class Worker
+    # Returns an array of all worker objects currently processing jobs.
+    def self.working
+      names = all
+      return [] unless names.any?
+
+      reportedly_working = data_store.working_keys(names)
+
+      reportedly_working.map do |key|
+        worker = find(key.sub("worker:", ''), :skip_exists => true)
+        worker.job = { 'queue' => key.split(':').last }
+        worker
+      end.compact
```

Comment:
Do we really need to `compact` here?  If `worker` were ever nil, then line 12 should break.

Reply:
I see that you're just mirroring the existing implementation, which I think makes sense to keep.